pandas/tests/series/methods/test_quantile.py

```diff
@@ -38,10 +38,11 @@ def test_quantile(self, datetime_series):
         result = Series([np.timedelta64("NaT")]).sum()
         assert result == pd.Timedelta(0)
 
+        s = Series(np.random.randn(100))
+        percentile_array = [-0.5, 0.25, 1.5]
         msg = "percentiles should all be in the interval \\[0, 1\\]"
-        for invalid in [-1, 2, [0.5, -1], [0.5, 2]]:
```

Comment:
Why are you deleting those?

Reply:
Should I need append the `Series` test ?